### DISCUSSION TITLE
automatic routing of links

### DISCUSSION THREAD

**Discussion by zairiint** [imixs/open-bpmn#263] [Q&A]
Does any one have an example of auto routing implementation with node glsp server

**Comment by rsoika** [comment]
As far as I know GLSP does not have such a feature. What exactly is your use case?

**Comment by zairiint** [comment]
My use case is to get an auto calculation of routing point of a link when linking tow node

**Reply by rsoika** [reply]
Do you mean a BPMN Link Event? In this case no rounting is necessary because the BPMN Link describes a association without a routing

![image](https://github.com/imixs/open-bpmn/assets/1339124/29d70e48-9023-4027-ac59-d234d79deda5)
